core/src/main/targetconfigs/vz/model.yaml

```diff
@@ -13,6 +13,10 @@ spec:
         # The WebLogic Domain Home
         domainHome: {{{domainHome}}}
 
+        # The domain home source type
+        # Set to PersistentVolume for domain-in-pv, Image for domain-in-image, or FromModel for model-in-image
+        domainHomeSourceType: {{{domainHomeSourceType}}}
```

Comment:
This requires a change to WebLogic Image Tool.  It will not break, but it will not be populated.